app/components/autologin-frame.js

```diff
@@ -16,6 +16,6 @@ export default Component.extend({
       return false;
     }
 
-    return typeof window.Cookies.get('autologin_done') === 'undefined' && this.currentUser.isAuthenticated;
+    return window.Cookies.get('autologin_done') === '1' && this.currentUser.isAuthenticated;
```

Comment:
I think this should be `window.Cookies.get('autologin_done') !== '1' && window.Cookies.get('autologin_done') !== '2'`
- If it's `'1'` then we already actively synced the cookies (or failed because of 3rd party cookies blocked, but there is no reason to try again)
- If it's `'2'` then we already passively synced the cookies which means there is access_token set on the other domain

Reply:
That was stupid change... fix in eafb74498